index.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const NR = require('node-resque');
+
+// SET UP THE CONNECTION
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: process.env.REDIS_HOST,
+    password: process.env.REDIS_PASSWORD,
+    port: process.env.REDIS_PORT,
+    database: 0
+};
+
+// DEFINE YOUR WORKER TASKS
+const jobs = {
+    start: {
+        /* config looks like this
+            executor:
+                name: k8s-vm
+                options:
+                    kubernetes: ...
+            buildConfig:
+                apiUri:
+                buildId:
+                container:
+                token:
+         */
+        perform: (config, callback) => {
+            // eslint-disable-next-line
+            const ExecutorPlugin = require(`screwdriver-executor-${config.executor.name}`);
```

Comment:
Added a check for this `screwdriver-executor-queue` so it doesn't loop forever. Renamed this repo to `screwdriver-queue-worker`, since it's not really an `executor` so we don't want to follow the executor convention.